apps/complaints/acceptance/pages/confirmation.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
 module.exports = {
-  url: 'confirmation'
+  url: 'confirmation',
+  content: {
+    sent: 'The complaint has been sent.',
```

Comment:
It's more consistent, as the content could change but the surrounding element is likely to remain unchanged